interview-questions.md

```diff
@@ -7,55 +7,56 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer:git is a open sourced version control system used by programmers.
+  git is the tool used to track and manage your code source history and github is the cloud base hosting service that lets you manage git repositories.
 
-  Researched answer:
+  Researched answer: Git is a version control system that lets you manage and keep track of your source code history. GitHub is a cloud-based hosting service that lets you manage Git repositories. If you have open-source projects that use Git, then GitHub is designed to help you better manage them
 
 
 
 2. What types of evaluations will return a Boolean value?
 
-  Your answer:
+  Your answer: true and false
 
-  Researched answer:
+  Researched answer: a binary variable, having two possible values called “true” and “false.”
 
 
 
 3. What is an index? What is the difference between index and value?
 
-  Your answer:
+  Your answer: Index is the address of a given character and value is the character in the address
 
   Researched answer:
-
+  An index is a numerical representation of an item's position in a sequence. This sequence can refer to many things: a list, a string of characters, or any arbitrary sequence of values. In some programming languages, an array can be defined by key-value pairs, where a specific key points to a value.
```

Comment:
https://en.scratch-wiki.info/wiki/Index

Reply:
An Idex is the location or address of an assigned variable or character, while an array stores or lists mulitple variables in one location.